api/config/routes.rb

```diff
@@ -91,14 +91,7 @@
     resources :states, only: [:index, :show]
 
     resources :taxonomies do
-      member do
-        get :jstree
-      end
-      resources :taxons do
-        member do
-          get :jstree
```

Comment:
Just saying, removing from the API might really upset anyone who wants to upgrade, but they're using this in their iOS or Android app. I know my previous employer [Kabuni](https://kabuni.com) is using it in their iOS app.